Polyfrost/Hytils-Reborn
# fix smaller bugs and move some regexes to LanguageData

## Description

- Move regexes to LanguageData. (origin #51)
- Thank Watchdog now works when some gets banned from a lobby
- Fix Player AdBlocker hides messages from yourself
- Fix Short Channel Names not working with Colored Friend Status (better solution)
- Fix Auto Friend only works with ranked players
- Fix Hide Ping in Tab is incompatible with Patcher's Number Ping

## Related Issue(s)

Fixes #11 

## How to test

- Turn on an affected regex features and see if it works
- Write spam messages (on your private SkyBlock island) and test if messages are hidden from chat
- Join a game with Patcher's Number Ping and Hide Ping in Tab activated

## Release Notes

```release-note
- Move some regexes to LanguageData, allowing for updates without needing to release a new version of Hytils
- Thank Watchdog now works when some gets banned from a lobby
- Fix Player AdBlocker hides messages from yourself
- Fix Short Channel Names not working with Colored Friend Status
- Fix Auto Friend only works with ranked players
- Fix Hide Ping in Tab is incompatible with Patcher's Number Ping
```
